DNN Platform/DotNetNuke.Abstractions/Skins/ISkinService.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Skins;
+
+using System.Collections.Generic;
+
+using DotNetNuke.Abstractions.Portals;
+
+/// <summary>Handles the Business Control Layer for Skins.</summary>
+public interface ISkinService
+{
+    /// <summary>Gets the folder name for the specified <paramref name="packageType"/>.</summary>
+    /// <param name="packageType">The type of the skin package.</param>
+    /// <returns>The folder name.</returns>
+    string GetFolderName(SkinPackageType packageType);
+
+    /// <summary>Gets the global default skin src.</summary>
+    /// <param name="packageType">The type of the skin package.</param>
+    /// <param name="skinType">The type of the skin.</param>
+    /// <returns>The global default edit skin.</returns>
+    string GetDefaultSkinSrc(SkinPackageType packageType, SkinType skinType);
+
+    /// <summary>Gets a skin package by its id.</summary>
+    /// <param name="packageId">The skin package id.</param>
+    /// <returns>The skin package.</returns>
+    ISkinPackageInfo GetSkinByPackageId(int packageId);
```

Comment:
Maybe this could be `GetSkinPackageById` to match what it returns and avoid confusion?

Reply:
Awesome! Thanks!